02_bubble/shuutech/main_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+)
+
+func TestBubble(t *testing.T) {
```

Comment:
You should have more test cases.

i.e. does your code handle zero length slice/array, 1-length, etc.

Reply:
I have added the additional test cases - please review and revert

Reply:
I have added test cases as per review